From cdfdb90e2d18978745f813ec1f16ca77154dc05d Mon Sep 17 00:00:00 2001
From: Hieu Huynh <hieu.huynh@amperecomputing.com>
Date: Fri, 24 Jun 2022 10:14:01 +0000
Subject: [PATCH 01/10] Allow user access from external repos

Add user_channel/user_mgmt.hpp to allow external application like
ampere-ipmi-oem to support some user access (create/delete) operations.

Signed-off-by: Hieu Huynh <hieu.huynh@amperecomputing.com>
---
 transporthandler.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/transporthandler.hpp b/transporthandler.hpp
index 165eeda..c943831 100644
--- a/transporthandler.hpp
+++ b/transporthandler.hpp
@@ -3,6 +3,7 @@
 #include "app/channel.hpp"
 #include "transportconstants.hpp"
 #include "user_channel/cipher_mgmt.hpp"
+#include "user_channel/user_mgmt.hpp"
 
 #include <ipmid/api-types.hpp>
 #include <ipmid/api.hpp>
-- 
2.34.1

